app/assets/javascripts/discourse/components/choose-topic.js.es6

```diff
@@ -4,14 +4,38 @@ import Component from "@ember/component";
 import discourseDebounce from "discourse/lib/debounce";
 import { searchForTerm } from "discourse/lib/search";
 import { observes } from "discourse-common/utils/decorators";
+import discourseComputed from "discourse-common/utils/decorators";
 
 export default Component.extend({
   loading: null,
   noResults: null,
   topics: null,
+  selectedTopic: null,
   selectedTopicId: null,
   currentTopicId: null,
-  topicTitle: null,
+  additionalFilters: "",
```

Comment:
It's slightly dangerous to initialize to `""` as all instances will point at the same string. In general with strings we replace them, but there are operations in Javascript that mutate them. Instead you should set these to `null` and initialize them in `init`